docs/appendices/release-notes/5.3.5.rst

```diff
@@ -47,6 +47,9 @@ See the :ref:`version_5.3.0` release notes for a full list of changes in the
 Fixes
 =====
 
+- Fixed an issue that resulted in hiding errors with the connection URL of
+  :ref:`CREATE SUBSCRIPTION<sql-create-subscription>`.
```

Comment:
I think we need to add that it's only for non-superusers. Superuser could see error AFAIK.